frontend/src/Components/StoreSearch/StoreCard.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import { Card, Button, Badge } from "react-bootstrap";
+
+const getOpenString = (hours) => {
+    const currentHour = new Date().getHours();
```

Comment:
Just a general thought about hours. Depending on what API we use to retrieve the store hours we should figure out a way to make sure that `new Date().getHours()` is using the same timezone as the `hours` string we pass in. Not something we have to address in this PR :) but I think a good thing to keep in mind as we start hooking up the backend

Reply:
This is a great point, I haven't considered this. [`getTimezoneOffset()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset) should come in handy